yarn.lock

```diff
@@ -14144,7 +14144,7 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-typescript@3.8.2, typescript@^3.8.2, typescript@^3.8.3:
+typescript@3.8.2, typescript@^3.6.0, typescript@^3.8.2, typescript@^3.8.3:
```

Comment:
I have no idea why I have this change, should I discard it? maybe I have some old version of typescript on my cloudtop?

Reply:
No, don't discard it. Let `yarn.lock` do what `yarn.lock` will